library/src/main/java/com/google/android/exoplayer/metadata/id3/APICFrame.java

```diff
@@ -0,0 +1,30 @@
+package com.google.android.exoplayer.metadata.id3;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+
+/**
+ * Created by apittenger on 5/11/16.
+ */
+
+public class APICFrame extends Id3Frame{
```

Comment:
Please use 2 character indentation to be consistent with the rest of ExoPlayer's codebase.